packages/forms-web-app/src/config.js

```diff
@@ -94,7 +94,7 @@ module.exports = {
 		stringValidation: {
 			listedBuildingNumber: {
 				regex: {
-					regex: '[0-9]',
+					regex: '^[0-9]+$',
```

Comment:
I'm gunna seem really picky now but Sonar recommends using the character classes rather than sets in cases like this, ie `/^\d+$/` 

<img width="799" alt="Screenshot 2023-11-01 at 12 38 15" src="https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/51144c7d-f12d-449f-a360-c73029a9b834">